packages/grz-pydantic-models/src/grz_pydantic_models/submission/metadata/v1.py

```diff
@@ -995,6 +995,27 @@ def check_duplicate_file_checksums(self):
 
         return self
 
+    @model_validator(mode="after")
+    def check_duplicate_file_paths(self):
+        file_paths = set()
+        for donor in self.donors:
+            for lab_datum in donor.lab_data:
+                if sequence_data := lab_datum.sequence_data:
+                    for file in sequence_data.files:
+                        file_path = file.file_path
+                        if not file_path in file_paths:
+                            file_paths.add(file_path)
+                        else:
+                            log.warning(
+                                f"Encountered duplicate file path '{file_path}' "
+                                f"in '{lab_datum.lab_data_name}' "
+                                f"in donor '{donor.donor_pseudonym}'. "
+                                f"Are you sure you want to add the same file path "
+                                f"multiple times? "
+                            )
```

Comment:
Remind me why this is a warning and not an error? Having trouble thinking of valid use-cases for this.

Reply:
Or well, this may make sense for BED files, which could very well be the exact same one for all donors.

Reply:
Ah, the BED file is a good point...